### PR TITLE
bots: Introduce rhel-7-8 image

### PR DESCRIPTION
**Pull request by marusak**
Lets see what it does on the first build

 * [x] image-refresh rhel-7-8

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh in progress on 4-cockpit-8.
Log: https://209.132.184.41:8493/logs/image-refresh-12655-20190829-061111/

**Comment by cockpituous** [comment]
image-refresh in progress on 3-cockpit-7.
Log: https://209.132.184.41:8493/logs/image-refresh-12655-20190829-061111/

**Comment by cockpituous** [comment]
Task failed: https://209.132.184.41:8493/logs/image-refresh-12655-20190829-061111/

**Comment by cockpituous** [comment]
Task failed: https://209.132.184.41:8493/logs/image-refresh-12655-20190829-061111/

**Comment by cockpituous** [comment]
image-refresh in progress on 1-cockpit-7.
Log: https://209.132.184.41:8493/logs/image-refresh-12655-20190829-061313/

File: bots/images/scripts/rhel-7-8.bootstrap [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -ex
+
+if [ -z "$SUBSCRIPTION_PATH" ] && [ -e ~/.rhel/login ]; then
+    SUBSCRIPTION_PATH=~/.rhel
+fi
+
+BASE=$(dirname $0)
+$BASE/virt-install-fedora "$1" x86_64 "http://download.redhat.com/rhel-7/nightly/RHEL-7/latest-RHEL-7.8/compose/Server/x86_64/os/" $SUBSCRIPTION_PATH
```
Comment (martinpitt) [inline bots/images/scripts/rhel-7-8.bootstrap]: It's download.devel.redhat.com.

**Comment by cockpituous** [comment]
Task failed: https://209.132.184.41:8493/logs/image-refresh-12655-20190829-061313/

**Comment by cockpituous** [comment]
image-refresh in progress on 1-ci-srv-05.
Log: https://209.132.184.41:8493/logs/image-refresh-12655-20190829-065959/

**Comment by cockpituous** [comment]
Task failed: https://209.132.184.41:8493/logs/image-refresh-12655-20190829-065959/

**Comment by cockpituous** [comment]
image-refresh in progress on 3-ci-srv-06.
Log: https://209.132.184.41:8493/logs/image-refresh-12655-20190829-091515/

**Comment by cockpituous** [comment]
image-refresh rhel-7-8 done: https://github.com/cockpituous/cockpit/commits/image-refresh-rhel-7-8-20190829-094141

**Comment by marusak** [comment]
Build passed. @martinpitt how is it with rhel-7.8 branch? Do you want to test it here? Should I test just `rhel-7-8` on master?

**Comment by martinpitt** [comment]
@marusak: cockpit master won't work on RHEL 7. So for this image, rhel-7-8@cockpit-project/cockpit/rhel-7.8 plus the usual batch of selenium/container/rhel-atomic seems right. Essentially, copy the rhel-7.7 testmap in the obvious manner.

**Comment by martinpitt** [comment]
rhel-7-8 failure:

    error: line 118: Unknown tag: Recommends: subscription-manager-cockpit

That's weird -- it's guarded by `%if 0%{?rhel} && 0%{?centos} == 0`, so somehow the rhel macro isn't defined properly?

**Comment by martinpitt** [comment]
Oh, nevermind -- that's the *master* branch, and that's known not to work. It needs to be the rhel-7-8@cockpit-project/cockpit/rhel-7.8 test indeed, as you already figured out :grin:

**Comment by martinpitt** [comment]
@marusak: Note that you still need to add rhel-7-8 test skips to all tests that are skipped on 7-7 (like check-kubernetes)

**Comment by marusak** [comment]
> as you already figured out

Heh, sorry, I triggered it by mistake

> Note that you still need to add rhel-7-8 test skips to all tests that are skipped on 7-7 (like check-kubernetes)

I don't see any rhel-7-7 skips on master. `rhel-7-8@cockpit-project/cockpit/rhel-7.8` takes tests from `rhel-7.8` branch, it needs to be addressed in another PR. 

`fedora-30/container-kubernetes` also does not exists on master, it should be `fedora-30/container-kubernetes@cockpit-project/cockpit/rhel-7.8`.

And not sure if `fedora-30/container-bastion` really tests anything with this image.

**Comment by martinpitt** [comment]
@marusak: Oh right(I still need to get used to testing non-master branches :grinning: ), let's fix the test skips on the rhel-7.8 branch then. I'm okay with landing that initial image, it doesn't break anything existing, after all.

> And not sure if fedora-30/container-bastion really tests anything with this image.

It doesn't for image refreshes indeed (and thus image-refresh doesn't run it). We just need to run it on changes to the rhel-7.8 branch, so testmap.py should have it.

Thanks!

**Comment by marusak** [comment]
46 tests failed for `rhel-7.8` branch. But there is not many things we can do about it in this PR.
@martinpitt good to lend?

**Comment by martinpitt** [comment]
> @martinpitt good to lend?

No, but it's fine to land it! :-)

Thanks!

**Pull request merged**